<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="GLWidget" name="glwidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionRest"/>
   </widget>
   <widget class="QMenu" name="menuRender">
    <property name="title">
     <string>Render</string>
    </property>
    <addaction name="actionRenderNormal"/>
    <addaction name="actionRenderCorners"/>
    <addaction name="actionRenderValence"/>
    <addaction name="actionRenderCurvature"/>
    <addaction name="separator"/>
    <addaction name="actionRenderOriginal"/>
    <addaction name="actionRenderSmoothed"/>
    <addaction name="actionRenderCollapsed"/>
   </widget>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="actionSmooth"/>
    <addaction name="actionEdgeCollapse"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
   <addaction name="menuRender"/>
   <addaction name="menuOperations"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRest">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionRenderNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionRenderCorners">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Corners</string>
   </property>
  </action>
  <action name="actionRenderCurvature">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Curvature</string>
   </property>
  </action>
  <action name="actionSmooth">
   <property name="text">
    <string>Smooth...</string>
   </property>
  </action>
  <action name="actionRenderOriginal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </action>
  <action name="actionRenderSmoothed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Smoothed</string>
   </property>
  </action>
  <action name="actionRenderCollapsed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Collapsed</string>
   </property>
  </action>
  <action name="actionEdgeCollapse">
   <property name="text">
    <string>Edge collapse...</string>
   </property>
  </action>
  <action name="actionRenderValence">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Valence</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
